web/scripts/e2e.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 server_pid=$!
```

Comment:
For this to run we need the local server running as we have it on `a11y.sh` script

Please, could you add these lines

```sh
yarn workspace web start &
yarn workspace web run wait-on http://localhost:3000
```